### PR TITLE
Emit only error logging during unittests

### PR DESCRIPTION
**Pull request by AndrejMitrovic**

### CODE REVIEW

File: source/agora/node/Node.d [hunk 1]
```diff
@@ -63,6 +63,7 @@ public class Node (Network) : API
         this.gossip = new GossipProtocol(this.network);
         this.exception = new RestException(
             400, Json("The query was incorrect"), string.init, int.init);
+        setLogLevel(config.logging.log_level);
```
Comment (Geod24) [inline source/agora/node/Node.d]: No!

**Review by Geod24 — CHANGES_REQUESTED** [review]
The `Node` constructor is not the place to do this. `setLogLevel` affects global data and there might be multiple instances of `Node`, two things which are essentially incompatible.

**Comment by AndrejMitrovic** [comment]
hmm.. damn global data. I forgot about that.

**Comment by AndrejMitrovic** [comment]
Updated. I've changed it to set the error level in `makeTestNetwork`, since it appears to be the entry-point to all networking-related unittests.

The calling code can then easily set it to something else via another `setLogLevel` call.

**Review by Geod24 — APPROVED** [review]

**Pull request merged**